lib/sync/util.js

```diff
@@ -7,7 +7,7 @@ var _ = require('underscore');
 var SYNC_LOGGER = 'SYNC';
 var loggers = {};
 
-var generateHash = function (plainText) {
+function generateHash(plainText) {
```

Comment:
@wei-lee what I was trying to do was not have anonymous functions. I guess that we could have named function expressions instead of function declarations. E.g. `var generateHash = function generateHash(plainText) {`.

Since the main purpose of this PR was to get rid of the try/catch, which actually seems necessary now, I'll just close the PR instead. :)